bin/nanocl/src/commands/stats.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{
+  config::CliConfig,
+  models::{ProcessStatsRow, StatsArg},
+  utils,
+};
+use futures::channel::mpsc;
+use futures::StreamExt;
```

Comment:
For the import the convention is installed crates first then local crates. also merge both `futures` together

Reply:
Merged the imports.